packages/react-mutation-mapper/src/component/gnomad/GnomadFrequency.tsx

```diff
@@ -146,7 +145,11 @@ export default class GnomadFrequency extends React.Component<GnomadFrequencyProp
                 display = <span>0</span>;
             } else {
                 // show frequency as number with 4 significant digits
-                display = <span>{significantDigits(result['Total'].alleleFrequency, 4)}</span>;
+                var significantDigitsFormatter = new Intl.NumberFormat("en", { 
```

Comment:
Yes, I think this is fine.  Support looks good.  However, I would subsistute the following https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/toLocaleString

It has the same options and doesn't confuse so much with "Intl" 

Also, it seems that passing the locale ("en") is optional.  If so, we should just not pass anything as we are not localized and we intend to use same formatting for any locale.

Reply:
@alisman thanks, one minor thing is that it looks like I have to pass a locale anyway, so I pass `undefined` there.